TinCan/RemoteLRS.cs

```diff
@@ -567,6 +584,7 @@ public StateLRSResponse RetrieveState(String id, Activity activity, Agent agent,
                 return r;
             }
             r.success = true;
+            r.content = state;
```

Comment:
The other methods similar to this one, `RetrieveAgentProfile` and `RetrieveActivityProfile`, suffer the same problem so it would be good to get them updated as well.